src/Encryption/composer.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name":        "cache/encryption-cache",
+    "type":        "library",
+    "description": "Add tag support to your PSR-6 cache implementation",
+    "keywords":    [
+        "cache",
+        "psr6",
+        "encrypted",
+        "encryption"
+    ],
+    "homepage":    "http://www.php-cache.com/en/latest/",
+    "license":     "MIT",
+    "authors":     [
+        {
+            "name":     "Aaron Scherer",
+            "email":    "aequasi@gmail.com",
+            "homepage": "https://github.com/aequasi"
+        },
+        {
+            "name":     "Tobias Nyholm",
+            "email":    "tobias.nyholm@gmail.com",
+            "homepage": "https://github.com/nyholm"
+        },
+        {
+            "name":     "Daniel Bannert",
+            "email":    "d.bannert@anolilab.de",
+            "homepage": "https://github.com/prisis"
+        }
+    ],
+    "require":     {
+        "php":                   "^5.5 || ^7.0",
```

Comment:
Lets support 5.5 if we have no reason to drop it. 

The current syntax (at least the one in the docs) are dubble pipe.